tests/unit/scala/test_utils_suite.py

```diff
@@ -423,3 +424,37 @@ def f():
         assert caplog.text.count("extra_warning_text") == 1
     finally:
         warning_dict.clear()
+
+
+def test_is_sql_select_statement():
+    select_sqls = [
+        "select * from dual",
+        "(select * from dual)",
+        "(((select * from dual)))",
+        " select * from dual",
+        "   select * from dual",
+        "( ( ( select * from dual",
+        "with t as (select 1) select * from t",
+        "(with t as (select 1) select * from t",
+        "(((with t as (select 1) select * from t",
+        " with t as (select 1) select * from t",
+        "   with t as (select 1) select * from t",
+        "( ( ( with t as (select 1) select * from t",
+        "select*fromdual",  # don't care if the sql is valid.
+        "SELECT 1" "SeLeCt 1" "WITH t as (select 1) select * from t",
```

Comment:
did you mean to have `"SELECT 1", "SeLeCt 1", "WITH t as (select 1) select * from t",` here? note the missing commas